opendebates/admin.py

```diff
@@ -103,3 +104,8 @@ class SiteModeAdmin(ModelAdmin):
     _missed_fields = set(_fields) - _used_fields
     if _missed_fields:
         fieldsets.append(('Other', {'fields': sorted(list(_missed_fields))}))
+
+    def reset_current_votes(self, request, queryset):
+        # Note, currently all submissions are under the one and only SiteMode.
+        models.Submission.objects.update(current_votes=0)
```

Comment:
I have no objection to doing it this way (in fact I like that it's more explicit), but the way I explained this to Lilia previously it happened as part of updating `previous_debate_time`. You or Katie will want to update her.

This also makes me wonder if we could support updating `previous_debate_time` retroactively, but after a few minutes of digging I couldn't find a way to update `current_votes` atomically in Django (based on `previous_debate_time`) without manually writing the subquery.